ios/ios/SplashView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SplashView.swift
+//  ios
+//
+//  Created by Thiên Nguyễn on 8/10/23.
+//  Copyright © 2023 orgName. All rights reserved.
+//
+
+import SwiftUI
+
+let SHOW_LOGO_DELAY = 0.5
+let SHOW_LOGO_DURATION = 1.0
+
+
+struct SplashView: View {
+
+    @State private var isLoaded = false
+
+    var body: some View {
+        ZStack {
+            Image("BackgroundSplash").resizable()
+                .aspectRatio(contentMode: .fill)
+
+            Image("LogoWhite")
+                .aspectRatio(contentMode: .fill)
+                .frame(maxWidth: .infinity)
+                .opacity(
+                    isLoaded ? 1.0 : 0.0
+                )
+        }
+        .ignoresSafeArea()
+        .animation(.easeIn(duration: SHOW_LOGO_DURATION).delay(SHOW_LOGO_DELAY), value: isLoaded)
+        .onAppear {
+            isLoaded.toggle()
+        }
+    }
+}
+
+struct SplashView_Previews: PreviewProvider {
+
+    static var previews: some View {
+        SplashView()
+    }
+}
```

Comment:
When you install the MacOS Beta with Xcode 15 beta, you can simple preview the view with Swift macro `#Preview`

```suggestion
#Preview {
    SplashView()
}
```